src/app/core/services/guided-tour/guided-tour-items.service.ts

```diff
@@ -0,0 +1,392 @@
+        route: 'projects/123ab4cd5e/dc/europe-west3-c/clusters/4k6txp5sq',
+        title: 'How Eventful',
+        text:
+          'The events of the machine deployments will be shown here. To avoid clutter we clear this list on every login.',
+        stepPosition: 'bottom',
+      },
+      {
+        id: 'km-gt-cluster-details-machine-deployments',
+        route: 'projects/123ab4cd5e/dc/europe-west3-c/clusters/4k6txp5sq',
+        title: 'Even More Data',
+        text: 'Click on the machine deployment to view the details of the underlying nodes.',
+        stepPosition: 'bottom',
+      },
+      {
+        id: 'km-gt-md-details-nodes',
+        route: 'projects/123ab4cd5e/dc/europe-west3-c/clusters/4k6txp5sq/md/first-machine-deployment',
+        title: 'The Last Layer',
+        text:
+          "Your machine deployment has a lot of information to discover too. To see even more, you can unfold a single node and check it's settings.",
+        stepPosition: 'bottom',
+      },
+      {
+        id: 'km-gt-md-details-edit',
+        route: 'projects/123ab4cd5e/dc/europe-west3-c/clusters/4k6txp5sq/md/first-machine-deployment',
+        title: 'Change the Settings',
+        text:
+          'You can edit machine deployment settings here. AAfter saving the changes, your old nodes will be replaced with nodes that use the updated settings.',
+        stepPosition: 'bottom',
+      },
+      {
+        id: 'km-gt-md-details-back-to-cluster',
+        route: 'projects/123ab4cd5e/dc/europe-west3-c/clusters/4k6txp5sq/md/first-machine-deployment',
+        title: 'A Step Back',
+        text:
+          'Go back to the cluster detail view here. To go back to the cluster overview or your projects, use the sidebar.',
+        stepPosition: 'bottom',
+      },
+      {
+        id: 'km-gt-help-menu',
+        title: "You're All Set!",
+        text:
+          'You can revisit the tour here. You can also find a link to the Docs, an FAQ and our support in this menu.',
+        stepPosition: 'bottom',
+      },
+    ];
+  }
+
+  guidedTourProject(): Project {
+    return {
+      creationTimestamp: new Date(),
+      id: '123ab4cd5e',
+      name: 'My first project',
+      status: 'Active',
+      owners: [
+        {
+          creationTimestamp: new Date(),
+          name: 'John Doe',
+          email: 'john.doe@example.com',
+        },
+      ],
+      clustersNumber: 0,
+    };
+  }
+
+  guidedTourDOCluster(): Cluster {
+    return {
+      creationTimestamp: new Date(),
+      id: '4k6txp5sq',
+      name: 'my-first-cluster',
+      spec: {
+        cloud: {
+          dc: 'do-fra1',
+          digitalocean: {
+            token: 'token',
+          },
+        },
+        version: '1.18.8',
+      },
+      status: {
+        url: 'https://4k6txp5sq.europe-west3-c.dev.kubermatic.io:30002',
+        version: '1.18.8',
+      },
+      type: ClusterType.Kubernetes,
+    };
+  }
+
+  guidedTourDOMachineDeployment(): MachineDeployment {
```

Comment:
This would need pretty high maintenance effort as we would have to keep it with sync all the time. I'd rather not keep data like this in a service.

Reply:
I am not sure if it wouldn't be better to operate on real data instead of mocks.

Reply:
What would happen if client doesn't have DigitalOcean enabled?

Reply:
We should definitely operate on real data and only guide user instead of doing actions for him. Inside the wizard, different customers might use different configurations and we should have a generic way to guide them through it.